src/main/java/seedu/module/model/module/TrackedModule.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Represents a TrackedModule in the ModuleList.
  */
-public class TrackedModule implements Module {
+public class TrackedModule implements ModuleTracked {
```

Comment:
I was actually thinking more of `TrackedModule implements Module, Trackable`, so that we could better separate the different behaviors between the ArchivedModule and TrackedModule. What are your thoughts?